src/main/resources/filter/web/content.filter

```diff
@@ -0,0 +1,3 @@
+# Filter file for recheck that will filter content
+
+attribute=text
```

Comment:
I don't quite get the commit message ... What are you trying to do?
As always ... We cannot just simply add `.filter`-files. They need to be registered.